src/Symfony/Component/Process/ProcessManager.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process;
+
+use SplQueue;
```

Comment:
I believe this should always be an `\SplQueue` usage and global namespaces are never important in Symfony (correct me if I'm wrong here)